mmcv/runner/hooks/ema.py

```diff
@@ -67,7 +67,7 @@ def after_train_iter(self, runner):
         for name, parameter in self.model_parameters.items():
             buffer_name = self.param_ema_buffer[name]
             buffer_parameter = self.model_buffers[buffer_name]
-            buffer_parameter.mul_(1 - momentum).add_(momentum, parameter.data)
```

Comment:
Is this modification necessary because of BC-breaking or what? The original implementation should be verified in some downstream tasks?